component/web/security/src/main/java/org/exoplatform/web/login/LoginServlet.java

```diff
@@ -171,6 +176,24 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         int status;
         if (req.getRemoteUser() == null) {
             if (username != null && password != null) {
+                // email authentication
+                if (username.contains("@")) {
```

Comment:
and what if username contains an @ ?

Reply:
basically, the username with @ is forbidden, but even if the username contains @ (with dirty creation), we will not find a user with email == username and the login will then be executed normally